cluster/gce/configure-vm.sh

```diff
@@ -242,13 +246,20 @@ EOF
 
 # This should only happen on cluster initialization
 function create-salt-auth() {
```

Comment:
That's actually not true. `create-salt-auth` requires KUBELET_TOKEN on first boot, but otherwise it doesn't actually generate it. I'll document the pre-rereqs.